src/frontend/packages/lib_classroom/src/components/DashboardClassroomStudent/index.spec.tsx

```diff
@@ -91,6 +91,7 @@ describe('<DashboardClassroomStudent />', () => {
       `${displayedStartingDate} - ${displayedStartingTime} > ${displayedEndingTime}`,
     );
     expect(classroomEnded).toHaveBeenCalledTimes(0);
+    expect(screen.getByText('Add to my calendar')).toBeInTheDocument();
```

Comment:
You should see this file https://github.com/openfun/marsha/blob/master/src/frontend/packages/lib_video/src/components/live/Student/StudentLiveStarter/StudentLiveAdvertising/index.spec.tsx#L55 to add ics related expects.